chmst/GSoC17_publishing_workflow
# Disable default switcher on edit state

### Summary of Changes
This PR disables the default switcher on edit state if the state is default. 


### Testing Instructions
Open a state which is default. 
Switch the default-switcher to "no".
Save the state.
You will get an Errormessage - see screen below, and the state is not saved.

Apply the patch and repeat the test.
You cannot switch to default "no". 

### Expected result
Expected is a warning that it is not possible to remove the default state. Because there must be a default state.
But it seems more convenient if the switch is disabled, So there is no need for warning - "don't make me think!".

Note: The switcher is disabled but at the moment it does not look disabled because at the  the switcher does not support the disabled attribute. This will be fixed soon. 

### Actual result
![default-switcher](https://user-images.githubusercontent.com/1035262/30253786-54b460c6-968c-11e7-8a3c-8c060c3ca1f2.PNG)


### Documentation Changes Required
No
